pkg/cloud/azure/services/virtualmachines/virtualmachines.go

```diff
@@ -346,3 +353,7 @@ func GenerateRandomString(n int) (string, error) {
 	}
 	return base64.URLEncoding.EncodeToString(b), err
 }
+
+func availabilitySetID(subscriptionID, resourceGroup, availabilitySetName string) string {
```

Comment:
maybe getAvailabilitySetID?

Reply:
it doesn't really get it from anywhere, the function generates it.